code/controllers/subsystem.dm

```diff
@@ -73,4 +73,10 @@
 
 //usually called via datum/subsystem/New() when replacing a subsystem (i.e. due to a recurring crash)
 //should attempt to salvage what it can from the old instance of subsystem
-/datum/subsystem/proc/Recover()
\ No newline at end of file
+/datum/subsystem/proc/Recover()
+
+//this is so the subsystem doesn't rapid fire to make up missed ticks causing more lag
+/datum/subsytem/on_varedit(editedvar)
+	if (edited_var == can_fire && can_fire)
```

Comment:
the branch name starts with patch, that means github editor, that means no, no testing 

=P